python/pyspark/pandas/data_type_ops/null_ops.py

```diff
@@ -43,6 +43,22 @@ class NullOps(DataTypeOps):
     def pretty_name(self) -> str:
         return "nulls"
 
+    def eq(self, left: IndexOpsLike, right: Any) -> SeriesOrIndex:
```

Comment:
@itholic are there a lot left? Can we fix them in batch?

Reply:
Yeah, this PR already include the same issues in batch. There is `CategoricalOps.eq` is left as a similar operation, but we need further investigation since it has different issues.